api/src/services/graphql/index.ts

```diff
@@ -177,6 +178,12 @@ export class GraphQLService {
 	getSchema(type: 'schema'): GraphQLSchema;
 	getSchema(type: 'sdl'): GraphQLSchema | string;
 	getSchema(type: 'schema' | 'sdl' = 'schema'): GraphQLSchema | string {
+		const key = `${type}_${this.accountability?.role}_${this.accountability?.user}`;
```

Comment:
Do we need to include the user in the key?

Reply:
Yes, as you can use `$CURRENT_USER` in permissions and for the upcoming policy structure where policies are attached to users as well 👍🏻